run_project.sh

```diff
@@ -0,0 +1,7 @@
+PROJECT_ROOT=`pwd`
```

Comment:
Thank you for this.

A minor thing: you need a `mix deps.get` at the top level before line 2 (he says from bitter experience :)